include/specDefWrapper.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+// this test is to not break pic-build or cmake builds
+#ifndef CMAKE_VERSION
+    #define CMAKE_VERSION=NO_CMAKE_IN_SPEC
+    #ifdef SPEC_CUDA
+        #define CUPLA_STREAM_ASYNC_ENABLED 1
+        #define ALPAKA_ACC_GPU_CUDA_ENABLED
+        #define ALPAKA_ACC_GPU_CUDA_ONLY_MODE
+    #elif defined SPEC_OPENMP
+        #define CUPLA_STREAM_ASYNC_ENABLED 0
+        #define ALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLED
+        // #define ALPAKA_ACC_CPU_B_SEQ_T_OMP2_ENABLED
```

Comment:
could you please remove this line.

Reply:
I think this line has some use for documentation, because it says we decided to go with the version above, but this would also work.

Reply:
I will merge it like it is. Note if you like to put something like this into the code you should explicitly mention how to use it.If it is only a piece of commented out code it looked like that someone missed to remove prototype code.